include/natalie/float_object.hpp

```diff
@@ -143,19 +129,19 @@ class FloatObject : public Object {
     Value to_f() { return this; }
     Value to_i(Env *) const;
     Value to_s(Env *) const;
-    Value truncate(Env *, Value) const;
+    Value truncate(Env *, Value);
```

Comment:
I hate that we lost these `const` qualifiers, but I know why it happened. Hoping someday I can figure out how to hydrate a Value without needing to remove `const` everywhere.... :thinking: